OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
+- rawlingsj
+- jstrachan
+- rajdavies
+- ankitm123
+- babadofar
+- tomhobson
+reviewers:
+- rawlingsj
+- jstrachan
+- rajdavies
+- ankitm123
+- babadofar
+- tomhobson
```

Comment:
Use this instead: https://github.com/jenkins-x-plugins/jx-pipeline/blob/main/OWNERS, add alias file from jx-pipeline repo as well.